include/RAJA/util/Operators.hpp

```diff
@@ -386,7 +386,7 @@ struct logical_and : public detail::comparison_function<Arg1, Arg2>,
   {
     return lhs && rhs;
   }
-  static constexpr const bool identity = true;
+  static constexpr bool identity() {return true;}
```

Comment:
I think I've only done that once.  Though I did set up and manage a webex for a meeting while physically in a different meeting another time...